gatsby/createPages.js

```diff
@@ -512,4 +511,58 @@ module.exports = exports.createPages = async ({ actions, graphql }) => {
             },
         })
     })
+
+    const LibraryTemplate = path.resolve(`src/templates/Library.tsx`)
```

Comment:
Honestly my reasoning was just semantic so if you feel like it's overkill then I can totally go back to how it was before. I felt like since these elements are basically 'built-in' to the library page and only the library page that they would make the most sense in the template. That was why I refactored out the Handbook template into it's own component to make it easier to create new templates with slightly altered layouts, but could totally understand that being overkill. Lmk what your feeling is :)

Reply:
Got it. Not gonna let it prevent a merge! There's always time to re-re-re-refactor. YOLO